frame/composable-traits/src/staking/rewards.rs

```diff
@@ -1,26 +0,0 @@
-use crate::time::{DurationSeconds, Timestamp};
```

Comment:
i think rewards should be separate config from staking as it can be used separately and diffent domain. SOLID engineering.

Reply:
I do not agree as the idea of the design is rewards pool is an inherent part of the staking pool, without rewards there is no staking. But also as explained in another comment, first priority is to get things functionally working according to the design, then we refactor things if it makes sense.